amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseRefresher.java

```diff
@@ -29,17 +29,31 @@
 
     /**
      * Creates the table that will store leases. Succeeds if table already exists.
-     * 
+     * Deprecated. Use createLeaseTableIfNotExists().
```

Comment:
You can use @link or @see javadoc annotation here to link people over to the new method.